From 4d0036a83ffb7d851510b3ca1545bcd6b6e4c734 Mon Sep 17 00:00:00 2001
From: Tianhao Chai <cth451@gmail.com>
Date: Tue, 12 Mar 2019 00:00:21 -0500
Subject: [PATCH] edit-dialog: Do not ellipsize label

The edit-dialog ui may ellipsize labels on the left side of the panel into
"..." when the label text length is longer than expected, which is the
case when working on ja_JP locale.

This commit sets `ellipsize` property of the labels to `none`.
---
 data/ui/edit-dialog.ui | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/data/ui/edit-dialog.ui b/data/ui/edit-dialog.ui
index 94f2ea44..53b49b27 100644
--- a/data/ui/edit-dialog.ui
+++ b/data/ui/edit-dialog.ui
@@ -144,7 +144,7 @@
                     <property name="xalign">1.0</property>
                     <property name="margin-top">12</property>
                     <property name="label" translatable="yes">Title</property>
-                    <property name="ellipsize">end</property>
+                    <property name="ellipsize">none</property>
                     <property name="sensitive" bind-source="GcalEditDialog" bind-property="writable" bind-flags="default" />
                     <signal name="size-allocate" handler="fix_reminders_label_height_cb" object="reminders_label" swapped="no" after="yes" />
                     <style>
@@ -162,6 +162,7 @@
                     <property name="xalign">1.0</property>
                     <property name="margin-top">12</property>
                     <property name="label" translatable="yes">Date</property>
+                    <property name="ellipsize">none</property>
                     <property name="sensitive" bind-source="GcalEditDialog" bind-property="writable" bind-flags="default" />
                     <style>
                       <class name="dim-label"/>
@@ -196,7 +197,7 @@
                     <property name="margin-top">12</property>
                     <property name="valign">start</property>
                     <property name="label" translatable="yes">Reminders</property>
-                    <property name="ellipsize">end</property>
+                    <property name="ellipsize">none</property>
                     <property name="sensitive" bind-source="GcalEditDialog" bind-property="writable" bind-flags="default" />
                     <style>
                       <class name="dim-label"/>
@@ -214,6 +215,7 @@
                     <property name="margin-top">12</property>
                     <property name="valign">start</property>
                     <property name="label" translatable="yes">Notes</property>
+                    <property name="ellipsize">none</property>
                     <property name="sensitive" bind-source="GcalEditDialog" bind-property="writable" bind-flags="default" />
                     <style>
                       <class name="dim-label"/>
@@ -230,6 +232,7 @@
                     <property name="xalign">1.0</property>
                     <property name="margin-top">12</property>
                     <property name="label" translatable="yes">Location</property>
+                    <property name="ellipsize">none</property>
                     <property name="sensitive" bind-source="GcalEditDialog" bind-property="writable" bind-flags="default" />
                     <style>
                       <class name="dim-label"/>
-- 
2.21.0

